cli/server/server.go

```diff
@@ -283,12 +283,14 @@ func startServer(ctx *cli.Context) error {
 	server := network.NewServer(serverConfig, chain)
 	rpcServer := rpc.NewServer(chain, cfg.ApplicationConfiguration.RPC, server)
 	errChan := make(chan error)
-	monitoring := metrics.NewMetricsService(cfg.ApplicationConfiguration.Monitoring)
+	prometheus := metrics.NewPrometheusService(cfg.ApplicationConfiguration.Prometheus)
+	pprof := metrics.NewPprofService(cfg.ApplicationConfiguration.Pprof)
```

Comment:
Is also helpful to have metrics of other commands (at least of `db restore`).

Reply:
`db dump` is the case as well, right?
And what kind of metrics are interesting? All?

Reply:
Figures out: db dump and restore could have pprof and it will help us.

Reply:
Time to update and merge this one.

Reply:
added metrics @fyrchik @roman-khimov